modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PureCloudDotNetGuestClientCodegen.java

```diff
@@ -0,0 +1,150 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.CodegenModel;
+import io.swagger.codegen.CodegenProperty;
+import io.swagger.codegen.SupportingFile;
+import io.swagger.models.Model;
+import io.swagger.models.Operation;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.util.*;
+
+public class PureCloudDotNetGuestClientCodegen extends CSharpClientCodegen {
```

Comment:
This should extend `PureCloudDotNetClientCodegen` so it doesn't have to duplicate code and future changes will apply to both SDKs. This class should only need to override the `getName` method unless there is something different about the guest SDK.